### PR TITLE
Opening Opensearch

### PR DESCRIPTION
**Pull request by scrawfor99**
Adds a proposal for more open repository maintenance and issue triaging.

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: OPENING_OPENSEARCH.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  
+## Feedback
+
+As with any proposal or project, it is important for iteration and improvement to take place so please use this opportunity to suggest changes or raise concerns related to the topics discussed above. 
+
+
+## Next Steps 
+
+The first concrete steps towards implementing these changes have taken place with advice having been gathered from other organizations who participate in an open source manner. 
+Advice has been received from members of the following groups regarding _opening_ OpenSearch Security: Jupyer; Amazon Chime SDK; OpenSearch Core; Amazon Linux; Amazon Product Marketing; AWS Scheduling; Amazon Open Source Observabilty.
```
Comment (DarshitChanpura) [inline OPENING_OPENSEARCH.md:59]: nit: `Jupyter`

**Review by DarshitChanpura — COMMENTED** [review]
Overall the changes look good. Maybe we can change the title to something along the lines of `COMMUNITY_ENGAGEMENT_GUIDELINES.md` or `COMMUNITY_INVOLVMENT.md`

**Comment by scrawfor99** [comment]
I think something may be wrong with the checks since I only edited a single file--a new markdown--and it is saying that it failed backward compatibility & plugin install.

**Comment by peternied** [comment]
@davidlago What do you think of this proposal, adding you as a reviewer

File: OPENING_OPENSEARCH.md (hunk 1 above)
Comment (DarshitChanpura) [reply OPENING_OPENSEARCH.md]: @scrawfor99 I don't see this change still for some reason

**Pull request closed without merging**